images/dockerregistry/Dockerfile

```diff
@@ -9,6 +9,9 @@ FROM openshift/origin-base
 RUN yum install -y tree findutils epel-release && \
     yum clean all
 
+# The registry doesn't require a privileged user.
+USER 1001
```

Comment:
Persistent storage access is based on the GIDs and SELinux context, assuming the storage plugin supports the ch* operations.  Based on the SCC used, the pod will either have it allocated to the SC for them or they will need to set it themselves if using a RunAs{Any|NonRoot} type of strategy.